### PR TITLE
AT-8782: Complete all functional requirements of this story except save to and load from snow

### PR DESCRIPTION
**Pull request by ajaynalamala-ccpo**

### CODE REVIEW

File: src/steps/02-EvaluationCriteria/JandA/DescriptionOfJustification.vue [hunk 1]
```diff
@@ -6,20 +6,155 @@
           <h1 class="mb-3">
             Let’s work on your justification for this exception to fair opportunity
           </h1>
-          <p>
-            Intro paragraph
+          <p v-if="selectedException === 'YES_FAR_16_505_B_2_I_B'">
```
Comment (ericyoungquist-ccpo) [inline src/steps/02-EvaluationCriteria/JandA/DescriptionOfJustification.vue:9]: No logic in markup. Please use getters.

File: src/steps/02-EvaluationCriteria/JandA/DescriptionOfJustification.vue [hunk 2]
```diff
@@ -6,20 +6,155 @@
...
+            Briefly describe how this action requires use of the “unique or highly
+            specialized capabilities” exception. Discuss the events, history, and
```
Comment (ericyoungquist-ccpo) [inline src/steps/02-EvaluationCriteria/JandA/DescriptionOfJustification.vue:11]: In all 3 cases, the vast majority of the text is exactly the same. Please use getters to fill in the blanks
```Briefly describe how this action requires use of the "_____" exception. Discuss the events, history, and circumstances that led to the current situation requiring use of procedures for other than fair opportunity competition. Your rationale should _____. ```

File: src/steps/02-EvaluationCriteria/JandA/DescriptionOfJustification.vue [hunk 3]
```diff
@@ -6,20 +6,155 @@
...
+            <ATATExpandableLink aria-id="ContactFAQ2">
+              <template v-slot:header>
+                What else should I include in my description of the justification?
+              </template>
+              <template v-slot:content>
+                <p>
+                  Be sure to explain any actual or apparent time lags between events.
+                  At the very least, include:
+                </p>
+                <ul>
```
Comment (ericyoungquist-ccpo) [inline src/steps/02-EvaluationCriteria/JandA/DescriptionOfJustification.vue:43]: Spacing is significantly off here (left is browser, right is Figma). Please add class "mb-4" to the `p` tag, and add class `_atat-ul` to the `ul` tag - it's a helper class we use to ensure correct list line spacing
Screen Shot 2023-04-24 at 7 53 19 PM

**Review by ericyoungquist-ccpo — CHANGES_REQUESTED** [review]

File: src/steps/02-EvaluationCriteria/JandA/DescriptionOfJustification.vue [hunk 4]
```diff
@@ -6,20 +6,155 @@
...
+  },
+})
+
+export default class DescriptionOfJustification extends Mixins(SaveOnLeave) {
+  justficationDescription = "";
+  selectedException = "";
+
+  private get currentData(): FairOpportunityDTO {
+    return {
+      // eslint-disable-next-line camelcase
```
Comment (ericyoungquist-ccpo) [inline src/steps/02-EvaluationCriteria/JandA/DescriptionOfJustification.vue:122]: remove this `eslint-disable` and one below - unnecessary here

File: src/steps/02-EvaluationCriteria/JandA/DescriptionOfJustification.vue [hunk 5]
```diff
@@ -6,20 +6,155 @@
...
+          <div v-else-if="selectedException === 'YES_FAR_16_505_B_2_I_A'">
+            <p>
+              Briefly describe how this action requires use of the “unusual and compelling
+              urgency” exception. Discuss the events, history, and circumstances that led
+              to the current situation requiring use of procedures for other than fair
+              opportunity competition. Your rationale should document the required delivery
+              schedule and lead-times involved, including a chronological explanation of events
+              that caused the exigent situation.
+            </p>
+            <ATATExpandableLink aria-id="ContactFAQ2">
```
Comment (ericyoungquist-ccpo) [inline src/steps/02-EvaluationCriteria/JandA/DescriptionOfJustification.vue:34]: Update ID to relevant to this file

**Review by tonysumner-ccpo — APPROVED** [review]

**Review by ericyoungquist-ccpo — APPROVED** [review]

**Pull request merged**